doc/md/fundamentals/4-control-flow/4-blocks.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 4
+---
+
+# Block expressions
+
+A block expression in Motoko is a sequence of statements enclosed in `{ ... }`. Blocks are used to group multiple operations, define local variables, and organize code for readability and logic.
```

Comment:
```suggestion
A block expression in Motoko is a sequence of declarations enclosed in `{ ... }`. 
Since every expression is also a declaration, the sequence can include expressions.
Blocks are used to group multiple operations, define local variables, and organize code for readability and logic.
```